pkgs/tools/misc/tmux-sessionizer/default.nix

```diff
@@ -5,20 +5,32 @@
 , openssl
 , pkg-config
 , Security
-}:
+, testers
+, tmux-sessionizer
+}: let
 
-rustPlatform.buildRustPackage rec {
-  pname = "tmux-sessionizer";
-  version = "0.2.3";
+  name = "tmux-sessionizer";
+  version = "0.3.0";
+  mainProgram = "tms";
+
+in rustPlatform.buildRustPackage {
+  pname = name;
+  inherit version;
```

Comment:
There seems to be a lot of formatting changes here, unrelated to the version bump. It would be nice if it could be split into two or potentially three commits, bumping the version, general cleanup and maybe adding yourself as a maintainer in the third one.

Reply:
Done :smile: 

Let's hope there's not too many more requests, because interactive staging is a bit of a pain with this PR...